core/common/test/format/DateTimeComponentsFormatTest.kt

```diff
@@ -298,14 +297,6 @@ class DateTimeComponentsFormatTest {
         )
 
         val incorrectUnparsableOffsets = listOf(
-            // Single non-digit characters
-            "a", "_", "+",
-            // Two characters: letter+digit, letter+symbol, digit+symbol
-            "a9", "y!", "1#",
```

Comment:
In general, I'd still expect both complete nonsense and, more importantly, plausible syntactic elements separating time zones from everything else not to get parsed: `}`, `)`, `]`, `!!!`, `#hi`, etc.